Source/Core/SlippiPlayback/SlippiPlayback.h

```diff
@@ -11,9 +14,29 @@ class SlippiPlaybackStatus
 	bool shouldJumpForward = false;
 	bool inSlippiPlayback = false;
 	volatile bool shouldRunThreads = false;
+	bool isHardFFW = false;
 	int32_t currentPlaybackFrame = INT_MIN;
 	int32_t targetFrameNum = INT_MAX;
 	int32_t latestFrame = -123;
-private:
+
+	std::thread m_savestateThread;
+	std::thread m_seekThread;
+
+	void startThreads(void);
+	void resetPlayback(void);
+	void prepareSlippiPlayback();
+
+  private:
+	void SavestateThread(void);
+	void SeekThread(void);
+	void processInitialState(std::vector<u8> &iState);
+
+	std::unordered_map<int32_t, std::shared_future<std::string>>
+	    futureDiffs;        // State diffs keyed by frameIndex, processed async
+	std::vector<u8> iState; // The initial state
```

Comment:
Needed to `#include "../Common/CommonTypes.h"` here in order to get this to build